doc/api/crypto.md

```diff
@@ -1214,6 +1214,10 @@ This can be called many times with new data as it is streamed.
 <!-- YAML
 added: v11.6.0
 changes:
+  - version: REPLACEME
+    pr-url: ???
```

Comment:
```suggestion
    pr-url: https://github.com/nodejs/node/pull/33360
```

Reply:
Only after submitting the PR, he/she knows what should URL to write down, 😄

Reply:
@Himself65 Yes, but it’s easy to forget.